transformers/dart-dvhcvn/lib/src/model.dart

```diff
@@ -62,10 +62,10 @@ class Level1 extends Entity<void, Level2> {
   void get parent => null;
 
   /// Finds sub-entity by ID.
-  Level2 findLevel2ById(String id) => findById<Level2>(children, id);
+  Level2? findLevel2ById(String id) => findById<Level2>(children, id);
```

Comment:
I think we can drop the type specifier here, Dart 2.12 is smart enough to figure this out. Something like this:

```
Level2? findLevel2ById(String id) => findById(children, id);
```